starter-code/routes/movie.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express')
+const router = express.Router()
+
+const Movie = require('../models/movie.models.js')
+
+
+router.get('/list', (req, res) => {
+
+    Movie.find()
+        .then(movies => res.render('movies', { movies }))
+        .catch(err => console.log('ERROR:', err))
+})
+
+
+router.get('/new', (req, res) => res.render('new-movie'))
+
+router.post('/new', (req, res) => {
+
+    const { title, genre, plot } = req.body
+
+    Movie.create({ title, genre, plot })
+        .then(() => res.redirect('/movies/list'))
+        .catch(err => console.log("ERRORR", err))
+})
+
+router.get('/delete', (req, res) => {
```

Comment:
```suggestion
router.get('/delete/:id', (req, res) => {
```
Falta el id que hace referencia a la entidad a eliminar